CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
+- Update OneSDK to v2.3.1 ([#30](https://github.com/superfaceai/one-service/pull/30))
```

Comment:
please make links to Prs consistent `- #30` vs `(#30)`. Either is fine by me, just consistent.